homeassistant/components/frontend/__init__.py

```diff
@@ -14,7 +14,7 @@
 from homeassistant.components import websocket_api
 from homeassistant.components.http.view import HomeAssistantView
 from homeassistant.config import async_hass_config_yaml
-from homeassistant.const import CONF_NAME, EVENT_THEMES_UPDATED
+from homeassistant.const import CONF_MODE, CONF_NAME, EVENT_THEMES_UPDATED
```

Comment:
The service schema should really not be using `CONF_` constants but `ATTR_` constants.